packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1344,6 +1380,11 @@ export default class NewVm extends BaseComponent {
         done={this._isDisksDone()}
       >
         <SectionContent column>
+          {this._srsNotOnSameHost() && (
```

Comment:
IMHO, it should have three different messages because it can have three different incompatibilities. Incompatibilities (1)  between selected SRs (2) between existing disks (3) between one existing disk on no shared SR and selected SRs.
Also, we need to find when display the message if it have incompatibilities between existing disks. And if it better to display the current message above or bellow the select because it's move it.

Reply:
My bad! the existing disks are displayed in the select disks.